BuckarooSdk/Services/VPay/VPayPayResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BuckarooSdk.Services.VPay
+{
+  public class VPayPayResponse : ActionResponse
+  {
+    public override ServiceEnum ServiceEnum => ServiceEnum.VPay;
+    public string CardExpirationDate { get; set; }
```

Comment:
Why is there an expirationdate for Vpay and not for Maestro? Seems to me that this should be a property for all credit / debit cards?

Reply:
It is for Maestro and other credit cards: as in PayResponse files.

Reply:
Maybe it is nice to combine all the credit / debit cards together, like it is on the buckaroo platform?

https://dev.buckaroo.nl/PaymentMethods/Description/creditcards#pay

Reply:
It was done in a way Buckaroo started (they created the SDK originally) - so each type has separate transaction type. Creating it other way seems for me to be beyond scope of this task (i wanted to use something to simplify stuff).